packages/fxa-auth-server/lib/sentry.js

```diff
@@ -88,6 +88,9 @@ async function configureSentry(server, config) {
       beforeSend(event, hint) {
         return filterSentryEvent(event, hint);
       },
+      integrations: [
+        new Sentry.Integrations.LinkedErrors({ key: 'jse_cause' }),
```

Comment:
I hadn't noticed before that the default Sentry LinkedErrors integration looks for `cause`. This updates it to look for the proper cause key for `verror`.